service/awsconfig/v4/templates/cloudconfig/small.go

```diff
@@ -0,0 +1,46 @@
+package cloudconfig
```

Comment:
So all this is only copy/paste. Due to the renaming and making the templates content of the constants the diff is not that useful.